### ISSUE TITLE
systems-actions no longer appear

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javamelody/javamelody#109]
```
Somewhere between .26 and .28 (I think in .27) my systems-actions disappeared 
(e.g.:
    View deployment descriptor 
MBeans MBeans     
View OS processes 
JNDI tree     
Opened jdbc connections
Database Database )

They show up in .26
My web.xml config for the filter:

    <filter>
        <description>Does monitoring through javamelody - see http://code.google.com/p/javamelody/wiki/UserGuide</description>
        <filter-name>javamelodyMonitoringFilter</filter-name>
        <filter-class>net.bull.javamelody.MonitoringFilter</filter-class>
        <init-param>
            <description>See http://code.google.com/p/javamelody/wiki/UserGuide, enables system cmds</description>
            <param-name>system-actions-enabled</param-name>
            <param-value>true</param-value>
        </init-param>
        <init-param>
            <description>Set this param to true to disable monitoring all together</description>
            <param-name>disable</param-name>
            <param-value>false</param-value>
        </init-param>
        <init-param>
            <description>Which metrics to display</description>
            <param-name>displayed-counters</param-name>
            <param-value>http,sql,error,log,jsp,spring,services</param-value>
        </init-param>
    </filter>

```

Original issue reported on code.google.com by `davidkar...@gmail.com` on 9 May 2011 at 8:31
- Blocked on: #147
- Merged into: #147
